packages/web-react/src/components/FieldGroup/stories/FieldGroup.tsx

```diff
@@ -0,0 +1,22 @@
+// Because there is no `dist` directory during the CI run
```

Comment:
I think that those comments are useless in this story.

Reply:
comments removed